recipes/libtike/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "libtike-cufft" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/carterbox/ptychocg/archive/{{ version }}.tar.gz
+  sha256: 2f4d2f32037ed69ceb19f8a2d8e70acf67c2f2747d38da6e5a2b766fc0f92fa3
+
+build:
+  number: 0
+  skip: True  # [py<35 or (not linux64) or cuda_compiler_version == "None"]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - swig
+  host:
+    - git
```

Comment:
Would move `git` to `build` as it likely only is used for the CLI (unless I'm missing something here).

Reply:
I don't think git is required at all. I think this may be leftover from earlier when I building from a git tag instead of an archive.